### PR TITLE
Implementing the MS Teams WebHook

### PR DESCRIPTION
**Pull request by rfelber**
This PR is implementing a new ReadOnlyHook to integrate MS Teams via WebHook. This hook can send a _finding summary_ message in the MessageCard Style.

### CODE REVIEW

File: hooks/teams-webhook/.helmignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+*.tmproj
+.vscode/
+# Node.js files
+node_modules/*
+package.json
+package-lock.json
+src/*
+config/*
+Dockerfile
+.dockerignore
```
Comment (nigthknight) [inline hooks/teams-webhook/.helmignore:30]: I would add a newline to make GH happy

File: hooks/teams-webhook/__testfiles__/messageCard.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            "name": "Show Results in Dashboard",
+            "targets": [
+                {
+                    "os": "default",
+                    "uri": "https://my-secureCodeBoc-dashboard.url/"
+                }
+            ]
+        }
+    ]
+}
```
Comment (nigthknight) [inline hooks/teams-webhook/__testfiles__/messageCard.json:69]: I would add a newline to make GH happy

File: hooks/teams-webhook/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (nigthknight) [inline hooks/teams-webhook/.gitignore:1]: I would add a newline to make GH happy

File: hooks/teams-webhook/.dockerignore [hunk 4]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (nigthknight) [inline hooks/teams-webhook/.dockerignore:1]: I would add a newline to make GH happy

**Comment by J12934** [comment]
Will merge in the other [repo](github.com/secureCodeBox/secureCodeBox) ;)

**Pull request closed without merging**